tempesta_fw/cache.c

```diff
@@ -436,6 +436,8 @@ tfw_cache_copy_resp(TfwCacheEntry *ce, TfwHttpResp *resp, TfwHttpReq *req,
 		n = field - resp->h_tbl->tbl;
 		/* Skip hop-by-hop headers. */
 		h = (n < TFW_HTTP_HDR_RAW && hbh_hdrs[n]) ? &empty : field;
+		/* Skip trailer fields from being saved (FIXME: #491). */
+		h = tfw_str_eolen(h) ? h : &empty;
```

Comment:
Trailing headers stays in the body. This change prevents them from being restored from the cache.